### PR TITLE
adding crn and com name to sendVariationForApprovalEmail to match the…

### PR DESCRIPTION
**Pull request by nishanthmahasamudram**
… new template

### CODE REVIEW

File: src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/LicenceService.kt [hunk 1]
```diff
@@ -373,6 +373,8 @@ class LicenceService(
           licenceId.toString(),
           updatedLicence.forename!!,
           updatedLicence.surname!!,
+          licenceEntity.crn!!,
+          licenceEntity.submittedBy?.username!!,
```
Comment (andrewrlee) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/LicenceService.kt:377]: think `licenceEntity` would have the previous details on? we only update a copy of the licence with the submitter.

 think `username` should be fine?

Comment (nishanthmahasamudram) [reply src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/LicenceService.kt:377]: Even the submitter is getting updated:
val username = SecurityContextHolder.getContext().authentication.name
    val submitter = communityOffenderManagerRepository.findByUsernameIgnoreCase(username)
      ?: throw ValidationException("Staff with username $username not found")

Comment (nishanthmahasamudram) [reply src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/LicenceService.kt]: updated.

**Review by andrewrlee — DISMISSED** [review]
LGTM

File: src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/LicenceService.kt [hunk 2]
```diff
@@ -401,6 +401,8 @@ class LicenceService(
           licenceId.toString(),
           updatedLicence.forename!!,
           updatedLicence.surname!!,
+          licenceEntity.crn!!,
```
Comment (andrewrlee) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/LicenceService.kt:404]: no big deal but could have used `updatedLicence `

Comment (nishanthmahasamudram) [reply src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/LicenceService.kt:404]: updated

**Review by andrewrlee — APPROVED** [review]
LGTM

**Pull request merged**